docs/installation.md

```diff
@@ -46,6 +46,27 @@ The recommended way to install Kapowarr is using Docker. After installing Kapowa
     4. Information on how to change the port can be found on the [Setup After Installation page](./setup_after_installation.md#port).
     5. Using a named volume in docker requires you to create the volume before you can use it (refer to [Named Volumes](#named-volumes))
 
+=== "Docker CLI (custom user)"
+    The command to get the docker container running can be found below:
+    ```bash
+    docker run -d \
+        --name kapowarr \
+        -e PUID:1000 \
+        - GUID:1000 \
```

Comment:
Should be `-e`

Reply:
I was partway through writing the amendment to that one already.